hammr/commands/image/image.py

```diff
@@ -108,6 +108,148 @@ def help_list(self):
+                table.add_row(["OS", appliance.distributionName + " " + appliance.archName])
+                table.add_row(["Template Id", appliance.dbId])
+                table.add_row(["Description", appliance.description])
+                return
+
+        scans = self.api.Users(self.login).Scans.Getall()
+        scans = scans.scans.scan
+        for scan in scans:
+            if parent_uri == scan.uri:
+                scanned_instances = self.api.Users(self.login).Scannedinstances.Getall()
+                scanned_instances = scanned_instances.scannedInstances.scannedInstance
+                for scanned_instance in scanned_instances:
+                    if scan.scannedInstanceUri == scanned_instance.uri:
+                        distro = scanned_instance.distribution
+                        table.add_row(["OS", distro.name + " " + distro.version + " " + distro.arch])
+                        table.add_row(["Scan Id", scanned_instance.dbId])
+                        return
+
+        my_software_list = self.api.Users(self.login).Mysoftware.Getall()
+        my_software_list = my_software_list.mySoftwareList.mySoftware
+        for my_software in my_software_list:
+            if parent_uri.startswith(my_software.uri):
+                container_templates = self.api.Users(self.login).Mysoftware(my_software.dbId).Templates.Getall()
+                container_templates = container_templates.containerTemplates.containerTemplate
+                for container_template in container_templates:
+                    if parent_uri == container_template.uri:
+                        distro = container_template.distribution
+                        table.add_row(["OS", distro.name + " " + distro.version + " " + distro.arch])
+                        table.add_row(["MySoftware Id", my_software.dbId])
+                        table.add_row(["Description", my_software.description])
+                        return
+
+    def do_info_draw_generation(self, info_image):
+        table = Texttable(0)
+        table.set_cols_align(["l", "l"])
+        table.header(["Status Details", ""])
+
+        if info_image.status.error:
+            status = "Error"
+        elif info_image.status.cancelled:
+            status = "Cancelled"
+        elif info_image.status.complete:
+            status = "Done"
+        else:
+            status = "Generating"
+        table.add_row(["Status", status])
+        if info_image.status.error:
+            table.add_row(["Error Message", info_image.status.message])
+            table.add_row(["Detailed Error Message", info_image.status.errorMessage])
+        else:
+            table.add_row(["Message", info_image.status.message])
+
+        print table.draw() + "\n"
+
+    def do_info_draw_publication(self, info_image):
+        pimages = self.api.Users(self.login).Pimages.Getall()
+        table = Texttable(0)
+        table.set_cols_align(["l", "l"])
+        table.header(["Published To Details", "Cloud Id"])
```

Comment:
`Published To Details` is not very meaningful, could you think about another title?